src/main/java/bookstore/security/CustomUserDetailsService.java

```diff
@@ -0,0 +1,20 @@
+package bookstore.security;
+
+import bookstore.repository.user.UserRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.stereotype.Service;
+
+@RequiredArgsConstructor
+@Service
+public class CustomUserDetailsService implements UserDetailsService {
+    private final UserRepository userRepository;
+
+    @Override
+    public UserDetails loadUserByUsername(String username) throws UsernameNotFoundException {
+        return userRepository.findByEmail(username).orElseThrow(
+                () -> new RuntimeException("Can't find user my email"));
```

Comment:
```suggestion
                () -> new UsernameNotFoundException("Can't find user my email"));
```
and remove it from the signature